### ISSUE TITLE
The Choice of 1e-4 Instead of 1e-6 in 'calculate_adaptive_weight'

### ISSUE THREAD

**Issue by WilliamLLee** [PKU-YuanGroup/WF-VAE#7]
- Description
In the calculate_adaptive_weight method, the denominator in the gradient normalization step adds a small constant 1e-4 to prevent division by zero or excessively small gradient norms. However, in the corresponding paper, a value of 1e-6 was used instead.

This discrepancy raises questions about the rationale behind choosing 1e-4 in the implementation. Was this adjustment made due to empirical observations, numerical stability concerns, or other factors?
```python 
def calculate_adaptive_weight(self, nll_loss, g_loss, last_layer=None):
    layer = last_layer if last_layer is not None else self.last_layer[0]

    nll_grads = torch.autograd.grad(nll_loss, layer, retain_graph=True)[0]
    g_grads = torch.autograd.grad(g_loss, layer, retain_graph=True)[0]

    d_weight = torch.norm(nll_grads) / (torch.norm(g_grads) + 1e-4)
    d_weight = torch.clamp(d_weight, 0.0, 1e4).detach()
    d_weight = d_weight * self.discriminator_weight
    return d_weight
```
- Question
1. Why was 1e-4 chosen instead of the 1e-6 used in the original paper? It seems that increasing the constant to 1e-4 results in a relatively smaller discriminator weight. Was this effect intentional, or could it potentially impact training dynamics?
2. Was this change made to address any specific numerical instability issues or observed behavior during training? Are there any empirical results or benchmarks supporting this adjustment?

**Comment by qqingzheng** [comment]
Apologize for the oversight in our code integration and release. This should be 1e-6. In the past, we intended to change this value to reduce the gradient in hopes of stabilizing the training, but we did not actually verify its effectiveness.

**Comment by qqingzheng** [comment]
Fixed
